common/static/common/js/karma.common.conf.js

```diff
@@ -269,8 +271,15 @@ function getBaseConfig(config, useRequireJs) {
     };
 
     var hostname = 'localhost';
+    var port = 9876;
     if (process.env.hasOwnProperty('BOK_CHOY_HOSTNAME')) {
         hostname = process.env.BOK_CHOY_HOSTNAME;
+        if (hostname === 'edx.devstack.lms') {
+            port = 19876;
```

Comment:
I think our main documentation on exposed ports is the `docker-compose.yml` file in the devstack repo, and I have comments next to the newly exposed ports in a PR that I'm preparing for that.  These port numbers also get printed to the console when starting the karma tests, which is why I changed them here instead of just mapping to different host ports in `docker-compose.yml`.